android/src/main/java/io/ably/flutter/plugin/AblyEventStreamHandler.java

```diff
@@ -0,0 +1,107 @@
+package io.ably.flutter.plugin;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import io.ably.lib.realtime.ChannelStateListener;
+import io.ably.lib.realtime.ConnectionStateListener;
+import io.flutter.plugin.common.EventChannel;
+
+
+public class AblyEventStreamHandler implements EventChannel.StreamHandler {
```

Comment:
Can I ask that we add some high level comments on what AblyEventStreamHandler does, even if just a one-liner for others who review this code in future?

Reply:
I think this applies generally where we have classes that serve a shared function, but don't map to Ably well known types

Reply:
Added docstring in d59f89a2e4a18ccdcef6443f45d90ac453507501

Reply:
@tiholic I've commented on that commit.